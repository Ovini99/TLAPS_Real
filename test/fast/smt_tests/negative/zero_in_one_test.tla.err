@!!BEGIN
@!!type:obligationsnumber
@!!count:0
@!!END

File "/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library/TLAPS.tla", line 1, character 1 to line 363, character 77:
[INFO]: All 0 obligation proved.
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligationsnumber
@!!count:1
@!!END

** Unexpanded symbols: ---

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:being proved
@!!prover:smt
@!!meth:time-limit: 5
@!!obl:
0 \in 1

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:failed
@!!prover:smt
@!!meth:time-limit: 5; time-used: 5.0 (100%)
@!!reason:timeout
@!!already:false
@!!obl:
0 \in 1

@!!END

Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:failed
@!!prover:zenon
@!!meth:time-limit: 10; time-used: 0.0 (0%)
@!!reason:false
@!!already:false
@!!obl:
0 \in 1

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:13:5:13:12
@!!status:failed
@!!prover:isabelle
@!!meth:auto; time-limit: 30
@!!reason:Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin

@!!already:false
@!!obl:
0 \in 1

@!!END

File "./zero_in_one_test.tla", line 13, characters 5-11:
[ERROR]: Could not prove or check:
           0 \in 1
File "./zero_in_one_test.tla", line 1, character 1 to line 15, character 4:
[ERROR]: 1/1 obligation failed.
There were backend errors processing module `"zero_in_one_test"`.
