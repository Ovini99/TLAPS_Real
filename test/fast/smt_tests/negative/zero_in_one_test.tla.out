
\* TLAPM version 1.5.0
\* launched at 2023-11-27 14:18:45 with command line:
\* /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../tlapm -I /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library --toolbox 0 0 --nofp zero_in_one_test.tla --stretch 1

(* created new ".tlacache/TLAPS.tlaps/TLAPS.thy" *)
(* fingerprints written in ".tlacache/TLAPS.tlaps/fingerprints" *)
(* will not use fingerprints (because of option `--nofp`), but will now load fingerprints from the file `.tlacache/zero_in_one_test.tlaps/fingerprints`, in order to overwrite with the new fingerprints, and then save the results at the end. *)
COMMAND-LINE ARGUMENT
file=.tlacache/zero_in_one_test.tlaps/tlapm_38e060.smt; PATH='/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin'; z3 -smt2 -v:0 AUTO_CONFIG=false smt.MBQI=true "$file" >.tlacache/zero_in_one_test.tlaps/tlapm_38e060.smt.out
CONTENT OF FILE SENT TO BACKEND
;; TLA+ Proof Manager 1.5.0
;; Proof obligation #1
;;   generated from file "./zero_in_one_test.tla", line 13, characters 5-11

(set-logic UFNIA)
(declare-sort u 0)
;; Standard TLA+ operators
(declare-fun int2u (Int) u)
(declare-fun u2int (u) Int)
(declare-fun tla__in (u u) Bool)

;; Terms, predicates and strings

(assert
  (forall ((X_3 Int)) (! (= (u2int (int2u X_3)) X_3) :pattern ((int2u X_3)))))
(assert
  (forall ((S_6 u) (T_7 u))
  (=> (forall ((X_3 u)) (= (tla__in X_3 S_6) (tla__in X_3 T_7))) (= S_6 T_7))))

;; Theorem to be Proved
(assert (not (tla__in (int2u 0) (int2u 1))))

(check-sat)
(exit)

Zenon error: exhausted search space without finding a proof
(* created new ".tlacache/zero_in_one_test.tlaps/zero_in_one_test.thy" *)
(* fingerprints written in ".tlacache/zero_in_one_test.tlaps/fingerprints" *)
